web/styles/popovers.css

```diff
@@ -336,19 +336,19 @@ ul {
     }
 }
 
-.popover-avatar {
-    height: 240px;
+.popover-inner {
     width: 240px;
```

Comment:
This change applies this `240px` maximum width to EVERY popover, not just this one. If you're going to make this change, you need to test every other popover in the app.

I think it's probably most correct to delete the ineffective `popover-inner` CSS, or move it to a scope like `.user-popover, user-info-popover .popover-inner` if we need it on that scope -- which would at least deserve an explanation for why it was needed, since I think we already have that width on the containing elements?

Reply:
@timabbott , thank you for catching that. I will be more careful in the future!
I have added two commits. One is to remove the invalid `popover-inner` and the other is to add width to prevent text overflow. I have also provided a detailed explanation in the commit message. Could you please review the pull request again? Thank you!